src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -1,5 +1,6 @@
 package games.strategy.engine.framework.startup.ui.editors;
 
+import com.google.common.base.Ascii;
```

Comment:
Checkstyle: This import is out of order.

Reply:
Thanks for location, fixed in: 8eadf5e